sigs.yaml

```diff
@@ -1663,6 +1663,31 @@ workinggroups:
     contact:
       slack: wg-cloud-provider
       mailing_list: https://groups.google.com/forum/#!forum/kubernetes-wg-cloud-provider
+    subprojects:
+    - name: aws-cloud-provider
+      owners:
+      - https://raw.githubusercontent.com/kubernetes/aws-cloud-provider/master/OWNERS
+    - name: azure-cloud-provider
+      owners:
+      - https://raw.githubusercontent.com/kubernetes/azure-cloud-provider/master/OWNERS
+    - name: cloudstack-cloud-provider
+      owners:
+      - https://raw.githubusercontent.com/kubernetes/cloudstack-cloud-provider/master/OWNERS
+    - name: gce-cloud-provider
+      owners:
+      - https://raw.githubusercontent.com/kubernetes/gce-cloud-provider/master/OWNERS
```

Comment:
gcp cloud provider instead of gce?

Reply:
+1. It should be GCP (based on the existing names conventions).